### PR TITLE
New Package python-Flask-0.10.1

### PR DESCRIPTION
**Pull request by pulux**
New Package python-Flask-0.10.1
New Package python-Werkzeug-0.9.6
New Package python-itsdangerous-0.24

### CODE REVIEW

File: srcpkgs/python-itsdangerous/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+maintainer="Markus Berger <pulux@pf4sh>"
+homepage="http://github.com/mitsuhiko/itsdangerous"
+license="BSD"
+distfiles="${PYPI_SITE}/i/itsdangerous/itsdangerous-${version}.tar.gz"
+checksum=cbb3fcf8d3e33df861709ecaf89d9e6629cff0a217bc2848f1b41cd30d360519
+
+python3.4-itsdangerous_package() {
+    noarch=yes
+    depends="python3.4"
+    replaces="python3.4-itsdangerous>=0"
```
Comment (xtraeme) [inline srcpkgs/python-itsdangerous/template]: What's the point of those `replaces`?

Comment (xtraeme) [inline srcpkgs/python-itsdangerous/template]: Those replaces are redundant and obviously wrong... please remove them.

File: srcpkgs/python-itsdangerous/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    pycompile_version="3.4"
+    pycompile_module="itsdangerous"
+    short_descr="${short_descr/Python2/Python3.4}"
```
Comment (xtraeme) [inline srcpkgs/python-itsdangerous/template]: It's `short_desc` not `short_descr`.

**Comment by pulux** [comment]
short_desc is fixes in 064413c 

The replaces statatement is inspired from python-Jinja2. Its also removed in 064413c

**Comment by xtraeme** [comment]
Those replaces are unnecessary, remove them.

**Pull request merged**